atlascope/core/job_types/nucleus_to_gland_distance.py

```diff
@@ -14,70 +17,108 @@
 schema = {
     "type": "object",
     "required": [],
-    "properties": {
-        "gland_map_path": {
-            "type": 'string',
-            "title": 'Path to gland map',
-            "default": 'atlascope/core/management/populate/inputs/gland_map.jpg',
-        }
-    },
+    "properties": {},
 }
 
 
+def generate_gland_map(input_image):
```

Comment:
Repeating myself but I'm not clear on why this function needs to exist. As for controlling randomness, if this does need to be here, I would make the RNG seed an optional argument with a default value.